src/stt-settings/credentials.js

```diff
@@ -1,28 +1,92 @@
-const fs = require('fs');
-const path = require('path');
-const electron = require('electron');
-const appUserDataPath = electron.remote.app.getPath('userData');
+const fs = require("fs");
+const path = require("path");
+const electron = require("electron");
+const downloadDeepSpeechModel = require("deepspeech-node-wrapper")
+  .downloadDeepSpeechModel;
+const appUserDataPath = electron.remote.app.getPath("userData");
+// TODO: consider moving deepspeech logic to a separate file from credentials.js?
+
+function getDeepSpeechModelFolderName( modelVersion = "0.6.0") {
+  return `deepspeech-${modelVersion}-models`;
+}
+
+function getDeepSpeechModelPath(deepspeechModelVersion) {
+  return path.join(appUserDataPath, getDeepSpeechModelFolderName(deepspeechModelVersion));
+}
+
+// TODO: add some way to check if model,
+// folder and necessary files,
+// are set in user folder in application libary path
+// Files required described in README of https://github.com/pietrop/deepspeech-node-wrapper
+function getIsDeepspeechModelSet() {
```

Comment:
```suggestion
function validateDeepSpeechModelPaths() {
```

You could throw an error - just my preference for errors for something that you expect to be there.